static/js/index.js

```diff
@@ -0,0 +1,42 @@
+const flaskapi_url = 'http://127.0.0.1:5000/fetchEmployee/'
+
+async function loadData(url) {
+    const response = await fetch(url)
+    let data = await response.json();
+    return data;
+}
+
+async function fetchEmployee(id){
+    const url_emp = flaskapi_url + id
+    data = await loadData(url_emp)
+    console.log("data from the backend is => ", data);
+    document.getElementById("firstName").value = data.firstName;
```

Comment:
You should also add a check here if the employee id searched was actually present in the DB or not. For this you can check if data is null or undefined or data.dob is undefined. You can raise an alert that Employee Id not found and not render anything.

Reply:
Sir can I use a try catch block here ? would that be a good way to handle this ?